apple/MarkdownTextInputDecoratorShadowNode.mm

```diff
@@ -0,0 +1,255 @@
+#include "MarkdownTextInputDecoratorShadowNode.h"
+
+#include <react/renderer/components/iostextinput/TextInputState.h>
+
+#include <React/RCTUtils.h>
+#include <react/renderer/components/view/conversions.h>
+#include <react/renderer/core/ComponentDescriptor.h>
+#include <react/renderer/textlayoutmanager/RCTAttributedTextUtils.h>
+#include <yoga/Yoga.h>
+
+#include "RCTMarkdownStyle.h"
+#include "RCTMarkdownUtils.h"
+
+namespace facebook {
+namespace react {
+
+extern const char MarkdownTextInputDecoratorViewComponentName[] =
+    "MarkdownTextInputDecoratorView";
+
+void MarkdownTextInputDecoratorShadowNode::initialize() {
+  ShadowNode::traits_.unset(ShadowNodeTraits::ForceFlattenView);
+}
+
+void MarkdownTextInputDecoratorShadowNode::overwriteMeasureContent() {
+  const auto &children = getChildren();
+  if (children.empty()) {
+    return;
+  }
+  react_native_assert(
+      children.size() == 1 &&
+      "MarkdownTextInputDecoratorView received more than one child");
+
+  const auto child = std::dynamic_pointer_cast<const TextInputShadowNode>(children[0]);
+  react_native_assert(
+      child != nullptr &&
+      "MarkdownTextInputDecoratorView received child other than a TextInput");
+
```

Comment:
Do we need to add `child->ensureUnsealed();` here?

Reply:
[e876440](https://github.com/Expensify/react-native-live-markdown/pull/566/commits/e8764409ca095a60e80f9d4a34fd7191a7f64ef2)